cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -392,47 +393,60 @@ share
   -- The table containing pools' on-chain reference to its off-chain metadata.
 
   PoolMetadataRef
-    poolId              PoolHashId
-    url                 Text
-    hash                ByteString          sqltype=hash32type
+    poolId              Types.PoolIdentifier      sqltype=text
+    url                 Types.PoolUrl             sqltype=text
+    hash                Types.PoolMetaHash        sqltype=text
     UniquePoolMetadataRef poolId hash
 
   -- The table containing the metadata.
 
   PoolMetadata
-    poolId              PoolHashId
-    tickerName          Text
-    hash                ByteString          sqltype=hash32type
-    metadata            Text
+    poolId              Types.PoolIdentifier      sqltype=text
+    tickerName          Types.TickerName          sqltype=text
+    hash                Types.PoolMetaHash        sqltype=text
+    metadata            Types.PoolMetadataRaw     sqltype=text
     pmrId               PoolMetadataRefId Maybe
     UniquePoolMetadata  poolId hash
 
+  -- The pools themselves (identified by the owner vkey hash)
+
```

Comment:
No, the existing `db-sync` already has tables that contain what you are putting in `Pool` and `RetiredPool`.

The tables are `PoolUpdate` and `PoolRetire`.